docs/source/quantization.rst

```diff
@@ -1009,6 +1009,9 @@ if ``dtype`` is ``torch.qint8``, make sure to set a custom ``quant_min`` to be `
 you call the `torch.ao.quantization.get_default_qconfig(backend)` or `torch.ao.quantization.get_default_qat_qconfig(backend)` function to get the default ``qconfig`` for
 ``x86`` or ``qnnpack`` backend
 
+2. If you are using the ``onednn`` backend, we will use 8 bits for activation in the default qconfig mapping ``torch.ao.quantization.get_default_qconfig_mapping('onednn')``
```

Comment:
English is not my native, but shouldn't documentation better be in passive tense
```suggestion
2. If ``onednn`` backend is selected, 8 bits for activation in the default qconfig mapping ``torch.ao.quantization.get_default_qconfig_mapping('onednn')`` will be used.
```

Reply:
Thanks for the suggestions. Update it.